README.md

```diff
@@ -176,6 +176,8 @@ wget -q -O - https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | bash
 curl -s https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | bash
 ```
 
+#### Create a cluster
```

Comment:
If the cluster is created by 
```
k3d cluster create test-cluster
```
The Nginx ingress controller will not be deployed. So maybe better to clarify that how one can create a cluster with a different name but still want to deploy the Nginx ingress controller. 

Or just remove it so that a new cluster should always be created by `make k3d-create-cluster`